frontend/views/containers/contributions/IncomeForm.vue

```diff
@@ -0,0 +1,219 @@
+<template lang='pug'>
```

Comment:
Sandrina's recent PR for the income details deleted this file. I don't think it should be re-added, instead her work needs to be used instead